### ISSUE TITLE
publishing non-matching names as verbatim?

### ISSUE THREAD

**Issue by claudenozeres** [iobis/non-matching-names#9]
Hello, I notice the non-matching names fall under 'scientific name', even for non-taxonomic terms like 'egg'. Could it be recommended to use the DarwinCore terms of _verbatimIdentification_ or _verbatimScientificName_, and are these available when publishing data to an OBIS IPT?

**Comment by pieterprovoost** [comment]
@claudenozeres Yes, of course those values should not be in `scientificName`, and depending on the case `verbatimIdentification` and/or `lifeStage` would be appropriate. But this code is only about validating `scientificName` as this is the field we use for taxon matching with WoRMS. Note that `verbatimScientificName` is not a Darwin Core term, although GBIF adds this field to store the scientific name as provided to GBIF. In OBIS this is called `originalScientificName` but it would make sense to rename this.

**Comment by claudenozeres** [comment]
Thanks @pieterprovoost. I was intially asking about these name fields because we discovered while mapping our data in OBIS Canada IPT that the Darwin Core term of verbatimIdentification is not available. So I was looking for alternatives, and came here to get clues, and thought it may be a potential issues in published names that could be raised (terms showing up in scientificName because people do not know or have alternatives). Our IPT also does not does not seem to have originalScientificName, nor verbatimScientificName.
